### ISSUE TITLE
EU Cyber Resilience Act

### ISSUE THREAD

**Issue by pcihon** [github/developer-policy#10]
GitHub Policy is working to represent the interests of developers with EU policymakers as they consider the Cyber Resilience Act. We filed [comment with the European Commission](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/13410-Cyber-resilience-act-new-cybersecurity-rules-for-digital-products-and-ancillary-services/F3376627_en) to propose several amendments that would improve the regulation. [Details available here](https://github.blog/wp-content/uploads/2023/03/GitHub_Position_Paper-Cyber_Resilience_Act.pdf), but in summary the amendments focus on:
1. Improve clarity of Recital 10 exemption for open source.
1. Exempt open source in Article text to improve certainty.
1. Clarify the intended scope of “Distributor” with regards to software development platforms.
1. Clarify scope for software components not intended for end-use.
1. Revise Annex I requirements to apply industry best practices.

We are collaborating with representatives across the open source ecosystem to inform policymakers as discussions on the Cyber Resilience Act move forward. Two such venues for getting involved (you can, too) are:

* [OpenForum Europe](https://openforumeurope.org/open-source/)
* [OpenSSF](https://openssf.org/getinvolved/)

In this issue, please share your thoughts on our amendments and ways to improve the Cyber Resilience Act.

**Comment by lfrancke** [comment]
I assume this can be closed now? If not it needs a signifcant update :)
GitHub is still engaging with OFE and other initiatives but the CRA is finished for now.

**Comment by felixreda** [comment]
Indeed! The GitHub Developer Policy team is engaged in policy discussions on secondary legislation such as the technical descriptions of critical and important products, or the upcoming Art. 26 (2) (a) guidance on how the CRA will be applied to open source actors. A longer description of the state of play and our priorities can be found on the [GitHub blog](https://github.blog/open-source/maintainers/what-the-eus-new-software-legislation-means-for-developers/).
